public/app/features/alerting/unified/components/rule-editor/labels/LabelsField.test.tsx

```diff
@@ -0,0 +1,181 @@
+import { render, screen, waitFor, within } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+import React from 'react';
+import { FormProvider, useForm } from 'react-hook-form';
+import { Provider } from 'react-redux';
+
+import { clearPluginSettingsCache } from 'app/features/plugins/pluginSettings';
+import { configureStore } from 'app/store/configureStore';
+
+import { mockAlertRuleApi, mockApi, setupMswServer } from '../../../mockApi';
+import { getGrafanaRule, labelsPluginMetaMock } from '../../../mocks';
+import { GRAFANA_RULES_SOURCE_NAME } from '../../../utils/datasource';
+
+import LabelsField, { LabelsWithSuggestions } from './LabelsField';
+
+const labels = [
+  { key: 'key1', value: 'value1' },
+  { key: 'key2', value: 'value2' },
+];
+
+const FormProviderWrapper = ({ children }: React.PropsWithChildren<{}>) => {
+  const methods = useForm({ defaultValues: { labels } });
+  return <FormProvider {...methods}>{children}</FormProvider>;
+};
+const SubFormProviderWrapper = ({ children }: React.PropsWithChildren<{}>) => {
+  const methods = useForm({ defaultValues: { labelsInSubform: labels } });
+  return <FormProvider {...methods}>{children}</FormProvider>;
+};
+
+function renderAlertLabels() {
+  const store = configureStore({});
+
+  render(
+    <Provider store={store}>
+      <LabelsField />
+    </Provider>,
+    { wrapper: FormProviderWrapper }
```

Comment:
How about:
```suggestion
    <FormProviderWrapper>
      <LabelsField />
    </FormProviderWrapper>,
    { wrapper: TestProvider }
```
This way we don't need to configure store